veps/sig-network/passt/passt-beta.md

```diff
@@ -0,0 +1,115 @@
+
+```go
+type InterfaceBindingMethod struct {
+	...
+	PasstBinding InterfacePasstBinding `json:"passtBinding,omitempty"`
+}
+
+type InterfacePasstBinding struct{}
+```
+> [!NOTE]
+> DeprecatedPasstBinding is still a member of the struct. It represents a previous generation of passt which has been deprecated in release 1.3.
+
+A new Feature Gate `PasstBinding` will be introduced, in Beta phase, to condition the use of this binding.
+
+### Validation Webhook
+`passtBinding` network binding will only be allowed for interfaces bound to `Pod` network or `multus` default network.
+In addition, the admitter will also validate enablement of the `PasstBinding` FG. 
+A validation will be implemented in the network binding admitter.
+
+### virt-controller
+250Mi of RAM will be added to the compute container of virt-launcher pods if VMI has passtBinding, as those are the overheads that passt requires (if all ports are forwarded).
+
+### virt-handler
+The passt CNI currently invokes 2 sysctl calls:
+- Allow ping group range for user 107
+- Allow listening on privileged ports starting from 0.
+
+Both of those calls will be implemented in virt-handler for passtBinding related pods. The nmState spec properties already exist in the [code](https://github.com/kubevirt/kubevirt/blob/release-1.6/pkg/network/setup/netpod/netpod.go#L319)
+for `DeprecatedPasstBinding`; they just need to be enabled for passtBinding.
+
+### virt-launcher
+The network generator will be enhanced to populate the DomainXML with the passt network interface. The code will be similar to the code that currently configures 
+the domain in the sidecar container.
+For example: this is the interface representation when no specific ports to forward are specified. 
+```xml
+    <interface type="vhostuser">
+      <source dev="eth0"/>
+      <model type="virtio-non-transitional"/>
+      <alias name="ua-passtnet"/>
+      <backend type="passt" logFile="/var/run/kubevirt/passt.log"/>
+      <portForward proto="tcp"/>
+      <portForward proto="udp"/>
+    </interface>
+```
+As in the current sidecar code, when Istio is enabled, its reserved ports will be marked for exclusion. 
+
+### passt-repair call modifications
+Currently, passt-repair is currently called during migration only if a network binding plugin is defined in the VMI spec, and the `passtIPStackMigration` FG is enabled.
+This condition will be modified such that passt-repair would only be called if the VMI has a `passtBinding` (core) interface binding.
+
+## Update/Rollback Compatibility
+
+The `passt` network binding plugin will be deprecated once the feature is GA.
+Once deprecated, it'll still be maintained for another 3 releases, per KubeVirt's deprecation policy.
+This means that the plugin code will continue to work at least until release 1.10 (assuming GA at 1.8). However, without seamless migration support.
+Existing users may retain existing passt plugin based VMs until then. Seamless migration functionality would be lost.
```

Comment:
"Seamless migration functionality would be lost." repeats "However, without seamless migration support.".